README.md

```diff
@@ -1 +1,30 @@
 # awesome-books
+
+Books list web application.
+
+This project is a basic website that allows users to add books to a list. The books list is store locally. So you can close your browser and when you open it again you will still have you books list.
+
+# Built With
+
+HTML
+CSS
+JS
+
+# Setup
+
+Please clone my repo at https://github.com/david63011/awesome-books.git
+
+# Author
+
+👤 David Underwood
+
+GitHub: @david63011
+Twitter: @david630
+Show your support
+Give a ⭐️ if you like this project!
+
+# License
+
+This project is MIT licensed.
+
```

Comment:
- [ ] Good job so far 👏. Your project works fine. However, you need to have good documentation too. In order to make your README professional, Microverse provided a [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that you are advised to use for all your project. Please customize it to your project and make sure that all the sections are covered 😉

Reply:
ok I updated it